kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,99 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-08T00:00:00Z
+---
+
+All requests made to the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) are [gRPC requests](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines).
+Sometimes, when these requests can't be completed, you'll see this particular error message: `Context: deadline exceeded`.
+Network interruptions, short timeouts, server overload, and Query errors are some of the reasons that can cause this error.
+
+The following sections discuss the nature of this error and how to troubleshoot it.
+
+#### Check your logs
+
+:::note
+
+Cloud users cannot access some of the logs needed to diagnose the source of the error.
+
+If you're using Temporal Cloud, check your Workflow History for errors, look for latencies and errors in your Cloud metrics, and create a [support ticket](https://support.temporal.io/) with Temporal providing these details.
```

Comment:
workflow history would not contain errors. best to omit that part imo. just say to open support ticket and dont think customers have full logs available that I am aware of yet (maybe they do and im wrong tho so worth checking on this maybe)